content-for/test-body-footer.html

```diff
@@ -60,7 +58,10 @@
        */
       afterRender: function() {
         runningAudit = true;
-        axe.a11yCheck('#ember-testing-container', axe.ember.testOptions, axe.ember.a11yCheckCallback);
+        return axe.run('#ember-testing-container', axe.ember.testOptions)
+          .then(axe.ember.a11yCheckCallback)
+          .catch(Ember.Test.adapter.exception)
+          .then(() => runningAudit = false);
```

Comment:
axe-core only uses ES6 Promises, which don't have `.finally` (yet, it's in proposal)